ProjectName/src/features/serviceMessages/components/ListMessages.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import {Text, StyleSheet, SafeAreaView, Pressable} from 'react-native';
 import {useAppDispatch} from '../../../store/hooks';
-import {deleteMes} from '../../../store/messagesSlice';
+import {deleteMes} from '../../../store/messagesSlice/messagesSlice';
```

Comment:
Try to not cut words, because it's much harder to grasp code produced in this way. 
Winnig several characters is nothing when you sarcifice readability.